lib/altapay/altapay-php-sdk/lib/helpers.php

```diff
@@ -18,7 +18,7 @@ function autoLoader($class, $dir = null)
     }
 
     $listDir = scandir(realpath($dir));
-    if (isset($listDir) && !empty($listDir)) {
+    if (!empty($listDir)) {
```

Comment:
We know that it exists from the previous line, so just check for truthy:
```suggestion
    if ($listDir) {
```